entities/params/templates.go

```diff
@@ -16,8 +17,9 @@ func (p *PostTemplate) TrimSpaces() {
 
 // PutTemplate represents request body for PUT /api/templates
 type PutTemplate struct {
```

Comment:
@Dzalevski @FilipNikolovski why cant we update the name parameter?

Reply:
We should also be able to update name parameter. See campaign implementation.